pkg/performanceprofile/utils/schedstat/schedstat_test.go

```diff
@@ -0,0 +1,187 @@
+package schedstat
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func TestParseDataGetCPUs(t *testing.T) {
+	testCases := []struct {
+		name          string
+		schedStatData string
+		expectedCPUs  []string
+	}{
+		{
+			name:          "empty",
+			schedStatData: "",
+			expectedCPUs:  []string{},
+		},
+		{
+			name:          "simple",
+			schedStatData: simpleSchedStat,
+			expectedCPUs: []string{
+				"cpu0",
+				"cpu1",
+				"cpu2",
+				"cpu3",
+				"cpu4",
+				"cpu5",
+				"cpu6",
+				"cpu7",
+			},
+		},
+	}
+	for _, tt := range testCases {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := ParseData(strings.NewReader(tt.schedStatData))
+			if err != nil {
+				t.Errorf("failure: unexpected error %v", err)
+			}
+			gotCPUs := got.GetCPUs()
+			if !reflect.DeepEqual(gotCPUs, tt.expectedCPUs) {
```

Comment:
does the order of the cpus matter here ?  will expectedCPUs be always sorted as well? meaning, do we need to enforce that before the comparison?

Reply:
yup, `GetCPUs` guarantees to return a sorted set

Reply:
OTOH I should sort `expectedCPUs` explicitely